app/views/users/show.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is the profile for a given a user</h1>
\ No newline at end of file
+<div class="center flex-column mt-3 mx-3">
+    <%= render partial: "shared/user", locals: {user: @user}%>
+    <div class="w-50 w-100-mobile mb-3 bg-light p-2 border">
+      <h3>Bio</h3>
+      <p><%= @user.bio %></p>
+    </div>
+    <div class="d-flex flex-column w-50 w-100-mobile">
+      <% @user.recent_posts.each do |post| %>
+      <div class="d-flex flex-column bg-light p-2 border mb-3">
+        <h5><%= post.title %></h5>
+        <p><%= post.text %></p>
```

Comment:
- [ ] There is no column called `author_id` in the posts table. Please refactor your schema and ensure that the posts table has a column called `author_id`, not `user_id`.

<img width="549" alt="image" src="https://user-images.githubusercontent.com/77190203/173191324-4740b186-61b4-46a2-bce7-10518a517942.png">